statsmodels/stats/tests/test_weightstats.py

```diff
@@ -725,12 +731,13 @@ def test(self):
             assert_allclose(zstat, tc.statistic, rtol=1e-10)
             assert_allclose(pval, tc.p_value, rtol=1e-10, atol=1e-16)
 
-            zstat, pval = d1.ztest_mean(value=tc.null_value,
-                                       alternative=alternatives[tc.alternative])
+            zstat,
+            pval = d1.ztest_mean(value=tc.null_value,
```

Comment:
don't separate tuple unpacking.
This causes a test failure and is incorrect.
It would need parenthesis, AFAIR, and that makes it ugly and less readable